updatehub.go

```diff
@@ -29,11 +29,11 @@ type UpdateHub struct {
 	store            afero.Fs
 	firmwareMetadata metadata.FirmwareMetadata
 	state            State
-	pollInterval     int
 	timeStep         time.Duration
 	api              *client.ApiClient
 	updater          client.Updater
 	reporter         client.Reporter
+	initialPolling   bool
```

Comment:
I understand that the initialPolling needs to be identified however this patch does not yet use it so please add it later.